assignment-client/src/octree/OctreeServer.cpp

```diff
@@ -280,10 +283,10 @@ OctreeServer::~OctreeServer() {
 void OctreeServer::initHTTPManager(int port) {
     // setup the embedded web server
 
-    QString documentRoot = QString("%1/resources/web").arg(QCoreApplication::applicationDirPath());
+    QString documentRoot = QString("%1/resources/web").arg(ServerPathUtils::getDataDirectory());
```

Comment:
I was worried we missed a migration here, but there is no content for the AC that's not generated a run time.